pnginfo.c

```diff
@@ -309,11 +309,12 @@ pnginfo_displayfile (char *filename, int extractBitmap, int displayBitmap, int t
   printf ("  FillOrder: msb-to-lsb\n  Byte Order: Network (Big Endian)\n");
 
   png_textp text;
-  int num_text, max_text;
+  int num_text;
+  num_text=png_get_text(png,info,&text,NULL);
```

Comment:
The style used in this code is to have whitespace around the "punctuation" here, but it seems like too small a nit given how long it took me to notice this review.